tools/utils/src/orders.ts

```diff
@@ -95,3 +96,27 @@ export async function signOrder(
     },
   }
 }
+
+export function getByLowestSenderAmount(
```

Comment:
function name change to sortBy or getOrderBy?

Reply:
Happy to change to e.g. `getOrderByLowestSenderAmount` for consistency and clarity.